### DISCUSSION TITLE
Git branching sometimes stuck in 'New Branch'

### DISCUSSION THREAD

**Discussion by rbur004** [pulsar-edit/.github#180] [General]
Not sure where to post this.

Every now and then, when I try to change branches, I get the 'New Branch' dialog, and can't change the branch (except in a terminal, with git checkout).  If I restart Pulsar, it resolves itself. I just updated the version, but it still present.  Running on a Mac, version 1.105.2023051615 arm64

![image](https://github.com/pulsar-edit/.github/assets/145761/0b185627-a30c-43ca-aa65-33e2b763f1b3)

**Comment by confused-Techie** [comment]
If you don't mind, would you mind posting this as an issue over on the [`github`](https://github.com/pulsar-edit/github) package repo? That way we can get everything needed to look at the issue, and the right people will see it?

But thanks a ton for reporting the issue to us!
